app/repositories/users.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Libus
+  module Repositories
+    class Users < Libus::Repository[:users]
+      commands :create, update: :by_pk, delete: :by_pk
+
+      def all
+        users.order(:name).to_a
+      end
+
+      def admin_by_id?(user_id)
+        by_role("admin_user").by_pk(user_id).to_a.first
+      end
+
+      def basic_user_by_id?(user_id)
+        by_role("basic_user").by_pk(user_id).to_a.first
+      end
+
+      def admin_by_email?(email)
+        by_role("admin_user").where(email: email).to_a.first
```

Comment:
two things with those methods with: `?`
1. methods ending with `?` should return boolean value
2. instead of `where` and converting to array, getting first record, you can use more `one` method and `one!`